code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1298,13 +1298,14 @@
 		to_chat(user, "<span  class='notice'>You stuff the [I] into the mouth of the [src].</span>")
 		qdel(I)
 		I = null //??
-		molotov = 1
-		flags ^= OPENCONTAINER
-		name = "incendiary cocktail"
-		smashtext = ""
-		desc = "A rag stuffed into a bottle."
+		var/obj/item/weapon/reagent_containers/food/drinks/dummy = /obj/item/weapon/reagent_containers/food/drinks/molotov
```

Comment:
You don't need to do type casting with `initial` here, since these variables are on your own type. You can just do `initial(molotov)` for example.

Reply:
Not sure of what you mean, do you mean I can remove the leftmost type in this line? Or that I can ignore the ```.thing``` syntax in the variable copying below?